modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/nowplaying/PlayerHeadingSection.kt

```diff
@@ -0,0 +1,327 @@
+                                color = MaterialTheme.theme.colors.playerContrast02,
+                                maxLines = 1,
+                                modifier = Modifier
+                                    .clickable { onPodcastTitleClick() },
+                            )
+                        }
+                    }
+
+                    if (state.isChaptersPresent) {
+                        state.chapterSummary.takeIf { it.isNotBlank() }?.let {
+                            Spacer(
+                                modifier = Modifier.height(4.dp),
+                            )
+
+                            TextH70(
+                                text = it,
+                                maxLines = 1,
+                                color = Color.White.copy(alpha = 0.7f),
+                            )
+                        }
+                    }
+                }
+
+                if (state.isChaptersPresent) {
+                    Column(
+                        horizontalAlignment = Alignment.CenterHorizontally,
+                    ) {
+                        ChapterNextButtonWithChapterProgressCircle(
+                            onClick = onNextChapterClick,
+                            enabled = !state.isLastChapter,
+                            alpha = if (state.isLastChapter) 0.5f else 1f,
+                        )
+
+                        Spacer(modifier = Modifier.height(4.dp))
+
+                        TextH70(
+                            text = state.chapterTimeRemaining,
+                            color = Color.White,
+                            modifier = Modifier
+                                .alpha(0.4f),
+                        )
+                    }
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun ChapterPreviousButton(
+    onClick: () -> Unit,
+    enabled: Boolean,
+    alpha: Float,
+) {
+    IconButton(
+        onClick = onClick,
+        enabled = enabled,
+        modifier = Modifier
+            .alpha(alpha),
+    ) {
+        Box(
+            contentAlignment = Alignment.Center,
+            modifier = Modifier
+                .clip(CircleShape)
+                .wrapContentSize(),
+        ) {
+            Icon(
+                painterResource(R.drawable.ic_chapter_skipbackwards),
+                tint = Color.White,
+                contentDescription = stringResource(LR.string.player_action_previous_chapter),
+            )
+        }
+    }
+}
+
+@Composable
+private fun ChapterNextButtonWithChapterProgressCircle(
+    onClick: () -> Unit,
+    enabled: Boolean,
+    alpha: Float,
+) {
+    val contentDescription = stringResource(LR.string.player_action_next_chapter)
+    IconButton(
+        onClick = onClick,
+        enabled = enabled,
+        modifier = Modifier
+            .alpha(alpha)
+            .clearAndSetSemantics { this.contentDescription = contentDescription },
```

Comment:
```suggestion
            .semantics { this.contentDescription = contentDescription },
```

| Before | After |
| --- | --- |
| ![Screenshot_20241126_134411](https://github.com/user-attachments/assets/b8542352-da2a-4aef-8c1a-11dae37dad3e) | ![Screenshot_20241126_134307](https://github.com/user-attachments/assets/cfb5a4e4-be5b-4694-a362-ec584de481a3) | 

In TalkBack it's missing the button label.